x-pack/solutions/observability/plugins/slo/public/pages/slo_details/components/slo_details.tsx

```diff
@@ -13,17 +13,22 @@ import { BurnRatePanel } from './burn_rate_panel/burn_rate_panel';
 import { EventsChartPanel } from './events_chart_panel/events_chart_panel';
 import { HistoricalDataCharts } from './historical_data_charts';
 import { SLODetailsHistory } from './history/slo_details_history';
-import { Overview } from './overview/overview';
+import { Definition } from './overview/definition';
```

Comment:
you can move eveyrthing related to "definition" tab under its own folder instead of keeping it under overview